tests/core/test.srt

```diff
@@ -18,4 +18,3 @@ I guess I was wrong.
 5
 00:03:20,476 --> 00:03:22,671
 There was no danger at all.
-
```

Comment:
Looks like this is causing a test failure. Might need to adjust either here or in the test.

Reply:
I am not sure I understand how to fix this:
```
2022-09-27T14:53:27.4605815Z File "tests/core/dune.inc", line 329, characters 0-145:
2022-09-27T14:53:27.4606506Z 329 | (rule
2022-09-27T14:53:27.4606873Z 330 |  (alias runtest)
2022-09-27T14:53:27.4666046Z 331 |  (package liquidsoap)
2022-09-27T14:53:27.4666332Z 332 |  (deps
2022-09-27T14:53:27.4666589Z 333 |   (:test_srt ./test.srt)
2022-09-27T14:53:27.4666880Z 334 |   (:parsesrt parsesrt.exe))
2022-09-27T14:53:27.4667187Z 335 |  (action (run %{parsesrt} %{test_srt})))
2022-09-27T14:53:27.4667537Z (cd _build/default/tests/core && ./parsesrt.exe test.srt)
2022-09-27T14:53:27.4667892Z Fatal error: exception Stdlib.Queue.Empty
2022-09-27T14:53:27.4668561Z Raised at Stdlib__Queue.peek in file "queue.ml", line 60, characters 11-22
2022-09-27T14:53:27.4669189Z Called from Srt_parser.parse in file "src/core/tools/srt_parser.ml", line 86, characters 12-24
2022-09-27T14:53:27.4669798Z Called from Dune__exe__Parsesrt in file "tests/core/parsesrt.ml", line 6, characters 10-33
```

From what I see above I understand that the problem comes from the parser not from a snapshot not being equal ? Shouldn't the srt_parser be able to work around this ?

Reply:
It should probably check for 1 blank line and maybe the EOF ?

Reply:
Let me see if I can fix this one.

Reply:
Should be fixed in `main` now.